docs/spec/manifest-v2-1.md

```diff
@@ -67,6 +69,10 @@ Manifest provides the base accessible fields for working with V2 image format
 	
    SchemaVersion is the image manifest schema that this image follows.
 	
+>**Note**: **`fsLayers`** and **`history`** have deep relationship. Their 
```

Comment:
each element in `history` describes the docker runtime configuration for the layer at the same index in `fsLayers`